src/app/Topology/Shared/Entity/EntityDetails.tsx

```diff
@@ -214,24 +229,110 @@ export const TargetDetails: React.FC<{
     ];
   }, [serviceRef]);
 
+  const onToggle = React.useCallback(() => setExpanded((v) => !v), [setExpanded]);
+
   return (
     <DescriptionList {...props} columnModifier={columnModifier}>
-      {_transformedData.map((d) => (
-        <DescriptionListGroup key={d.key}>
-          <DescriptionListTermHelpText>
-            <Popover headerContent={d.helperTitle} bodyContent={d.helperDescription}>
-              <DescriptionListTermHelpTextButton>{d.title}</DescriptionListTermHelpTextButton>
-            </Popover>
-          </DescriptionListTermHelpText>
-          <DescriptionListDescription style={{ userSelect: 'text', cursor: 'text' }}>
-            {d.content}
-          </DescriptionListDescription>
-        </DescriptionListGroup>
-      ))}
+      {_transformedData.map(mapSection)}
+      <ExpandableSection
+        toggleText={isExpanded ? 'Show less' : 'Show more'}
+        onToggle={onToggle}
+        isExpanded={isExpanded}
+      >
+        <MBeanDetails isExpanded={isExpanded} connectUrl={serviceRef.connectUrl} />
+      </ExpandableSection>
     </DescriptionList>
   );
 };
 
+const MBeanDetails: React.FC<{ isExpanded: boolean; connectUrl: string }> = ({ isExpanded, connectUrl }) => {
+  const context = React.useContext(ServiceContext);
+  const [dayjs, dateTimeFormat] = useDayjs();
+  const addSubscription = useSubscriptions();
+  const [mbeanMetrics, setMbeanMetrics] = React.useState({} as MBeanMetrics);
+
+  React.useEffect(() => {
+    if (isExpanded) {
+      addSubscription(
+        context.api
+          .graphql<MBeanMetricsResponse>(
+            `
+            query MBeanMXMetricsForTarget($connectUrl: String) {
+              targetNodes(filter: { name: $connectUrl }) {
+                mbeanMetrics {
+                  runtime {
+                    startTime
+                    vmVendor
+                    vmVersion
+                  }
+                  os {
+                    version
+                    arch
+                    availableProcessors
+                  }
+                }
+              }
+            }`,
+            { connectUrl }
+          )
+          .pipe(map((resp) => resp.data.targetNodes[0].mbeanMetrics))
+          .subscribe(setMbeanMetrics)
+      );
+    }
+  }, [isExpanded, addSubscription, connectUrl, context.api, setMbeanMetrics]);
+
+  const _collapsedData = React.useMemo(() => {
+    return [
+      {
+        key: 'Start Time',
+        title: 'Start Time',
+        helperTitle: 'Start Time',
+        helperDescription: 'The time when this JVM process started.',
+        content: dayjs(mbeanMetrics?.runtime?.startTime || 0)
```

Comment:
We should handle cases where value is negative? I am not quite what is the meaning behind negative values..

![Screenshot from 2023-03-13 22-39-06](https://user-images.githubusercontent.com/68053619/224878688-2ae59385-182b-40bb-a1ff-7b0bd5ce16d3.png)

Reply:
I think that if this value is actually what the MBean interface reports, it should be displayed as-is. It looks odd, but the various fields may have different meanings for negative values. Usually it means something about the JVM or the host OS not supporting that kind of performance metric, which would be useful for the user to know. It may be the case that specific values have specific meanings so I would rather not try to hide the the value with some "friendlier" UI label.

Reply:
Oh okay right! That make sense!

Reply:
Oh yes that was the agent on `9988`!